### PR TITLE
Fix with paddings and margins not being applied responsively as well as max-width calculations with margins

### PR DESCRIPTION
**Pull request by adamduren**

### CODE REVIEW

File: src/core/style/layout.scss [hunk 1]
```diff
@@ -144,6 +104,22 @@
     $flexName: 'flex-#{$name}';
   }
 
+  @if $name == sm {
+    $mutiplier: 2;
+  } @else if $name == gt-sm {
```
Comment (dschwartz225) [inline src/core/style/layout.scss]: Why isn't this just

```
@else {
  multiplier: 1;
}
```

What's with all the else-ifs that all do the same thing? Just to be explicit.

File: src/core/style/layout.scss [hunk 2]
```diff
@@ -27,46 +27,6 @@
   flex-direction: row;
 }
 
-[layout-padding] > [flex-sm],
```
Comment (dschwartz225) [inline src/core/style/layout.scss:30]: So if I understand correctly, this block that you've removed was the original attempt at adjusting the padding/margin for responsive layouts, and it just failed miserably - what you've added below addresses it properly.

File: src/core/style/layout.scss [hunk 3]
```diff
@@ -144,6 +104,22 @@
     $flexName: 'flex-#{$name}';
   }
 
+  @if $name == sm {
+    $mutiplier: 2;
```
Comment (dschwartz225) [inline src/core/style/layout.scss:108]: Why is the multiplier 2 for small sizes only?

File: src/core/style/layout.scss (hunk 1 above)
Comment (adamduren) [inline src/core/style/layout.scss]: I need to revisit the math I think some of these should be 0.

**Pull request merged**